maps/perseus.map

```diff
@@ -63848,11 +63840,11 @@ shared_ex/base1a
 ( 64 1360 464 ) ( -64 1368 464 ) ( -64 1360 464 ) common/caulk 0 0 0 0.25 0.25 0 0 0
 }
 {
-( 904 -192 128 ) ( 904 -224 128 ) ( 872 -192 128 ) perseus_ex/base1a_nonsolid 256 192 0 -0.25 0.25 0 0 0
```

Comment:
I recommend you to duplicate `textures/perseus_ex/base1a_nonsolid` as `textures/perseus_ex/base1a` in `scripts/perseus_ex.shader` and use `perseus_ex/base1a` instead.

Reply:
tbh what was there previously looks like a mistake. There were two brush with similar looks nut differently-sized features so I just replaced that to use one brush with a floor grid texture from another part of the map.

It really just looked like the texture choice wasn't intentional

Reply:
okok